src/index.html

```diff
@@ -7,11 +7,189 @@
       content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
     >
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
-    <title>HTML Form</title>
+    <title>Layout HTML Form</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post"
+    >
+      <fieldset class="form__group">
+        <legend>Personal information</legend>
+        <div class="label__container">
+          <label for="surname">Surname:
+            <input
+              type="text"
+              id="surname"
+              name="surname"
+              autocomplete="off"
+              required
+              pattern="^[a-zA-Z]+$"
+            >
+          </label>
+        </div>
+        <div class="label__container">
+          <label for="name">Name:
+            <input
+              type="text"
+              id="name"
+              name="name"
+              autocomplete="off"
+              required
+              pattern="^[a-zA-Z]+$"
+            >
+          </label>
+        </div>
+        <div class="label__container">
+          <label for="age">How old are you?
+            <input
+              type="number"
+              id="age"
+              name="age"
+              autocomplete="off"
+              required
+              min="1"
+              max="100"
+              value="12"
+            >
+          </label>
+        </div>
+        <div class="label__container">
+          <label for="date-of-birth">Full date of birth:
+            <input
+              type="date"
+              id="date-of-birth"
+              name="date-of-birth"
+              autocomplete="off"
+              required
+            >
+          </label>
+        </div>
+        <div class="label__container">
+          <label for="terms">I accept the term of the agreement
+            <input
```

Comment:
Check list p2, fix in all cases

```suggestion
          <label for="terms">
            I accept the term of the agreement
            <input
```